### PR TITLE
Integrate pages into global searches

### PR DESCRIPTION
**Pull request by CallumWigg**
- Replace notion logo with higher quality and png background.
- Pages now show up on global searches without needing to tab through "Notion: Find Pages".

### CODE REVIEW

**Comment by wolloda** [comment]
Hello,

having pages show in global searches is a good idea. I think it should be optional (configurable in a [/src/notion.ini](../blob/master/src/notion.ini) file) though, I am personally inclined to `no` as a default.

**Comment by CallumWigg** [comment]
Hi @wolloda, a great suggestion. I've implemented the logic as recommended in the latest commit.

File: src/notion.py [hunk 1]
```diff
@@ -151,7 +157,7 @@ def _read_config(self):
         self._NOTION_SECRET = settings.get("notion_secret", "var", unquote=True)
         self._MATCH_PARENTS = settings.get_bool("show_parent_page_name", "main", True)
         self._SKIP_UNTITLED_PAGES = settings.get_bool("skip_untitled_pages", "main", True)
-
+        self._SEARCH_MODE = settings.get_bool("global_results", "main", True)
```
Comment (wolloda) [inline src/notion.py:162]: Defaults should be `False`.

**Review by wolloda — CHANGES_REQUESTED** [review]
Please change the defaults for the config item.

**Comment by wolloda** [comment]
Hello @CallumWigg, thanks for implementing the config option.

Please make it `False` by default and mention the option in the [README](https://github.com/wolloda/keypirinha-notionSearcher/blob/main/README.md#options)

**Pull request merged**